src/libguestfish.sh

```diff
@@ -15,6 +15,37 @@ if [ -z "${LIBGUESTFS_MEMSIZE:-}" ]; then
     # only if we detect this case, but eh.
     export LIBGUESTFS_MEMSIZE=2048
 fi
+# Get target base architecture
+basearch=$(python3 -c '
+import gi
+gi.require_version("RpmOstree", "1.0")
+from gi.repository import RpmOstree
+print(RpmOstree.get_basearch())')
+
+tmpd=$(mktemp -tdp /tmp gf-vsmt.XXXXXX)
+if [ "$basearch" = "ppc64le" ] ; then
+    qemu_wrapper=${tmpd}/qemu-wrapper.sh
+
+	cat <<-'EOF' > "${qemu_wrapper}"
+	#!/bin/bash -
+	i=0
+	while [ $# -gt 0 ]; do
+		case "$1" in
+		-machine)
+			shift 2;;
+		*)
+			args[i]="$1"
+			(( i++ ))
+			shift ;;
+		esac
+	done
+	exec qemu-system-ppc64 -machine pseries,accel=kvm:tcg,vsmt=8 "${args[@]}"
```

Comment:
Instead of reconstructing it, would it work to just let the wrapper do:

```
exec qemu-system-ppc64 "$@" -machine pseries,accel=kvm:tcg,vsmt=8
```

so that the `machine` argument is last?